cmake/inputs/patches/ep_absl/abseil-patch.cmake

```diff
@@ -0,0 +1,41 @@
+# ~~~
```

Comment:
Why is this patch here, needs documentation?
Why is it implemented with a cmake script, rather than as a patch file?